tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/types/ImportUserDefinedTypeTest.java

```diff
@@ -0,0 +1,34 @@
+package org.ballerinalang.test.bala.types;
+
+import org.ballerinalang.test.BCompileUtil;
+import org.ballerinalang.test.CompileResult;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+
+/**
+ * Test cases for import user defined type.
+ */
+public class ImportUserDefinedTypeTest {
```

Comment:
I don't think this name properly reflects what we are testing. There are other tests that (or rather pretty much all tests) import modules and use types defined in them.